app/validators/date_validator.rb

```diff
@@ -50,6 +53,15 @@ def illegal_year?(value)
     false
   end
 
+  def future_date?(value)
```

Comment:
This doesn't make sense having a validator that has a 'past' argument that expects the caller to specify -> { Time.zone.today } - that could be hard coded and used when 'past' is set to false.  
If the validator is actually validating against a user provided date / time then maybe it should be called 'min' or 'max' instead of 'past'.  Makes more sense to the caller